meta-intel-edison-bsp/recipes-kernel/linux/linux-yocto_4.16.0.bb

```diff
@@ -5,26 +5,31 @@ KBRANCH ?= "standard/base"
 
 require recipes-kernel/linux/linux-yocto.inc
 
-PV = "4.15.0"
+PV = "4.16.0"
 
 # this branch now contains both non-acpi kernel and acpi enabled kernel on top
-SRC_URI = "git://github.com/htot/linux.git;protocol=https;branch=eds-4.15.0-unified \
-        file://serial-8250-Don-t-service-RX-FIFO-if-interrupts-are-disabled.patch \
-        file://0001-perf-tools-Fix-compile-error-with-libunwind-x86.patch \
-	file://0001-Bluetooth-hci_serdev-Init-hci_uart-proto_lock-to-avo.patch \
-        file://usb_phy.cfg \
-        file://usb_dwc3.cfg \
+SRC_URI = "git://github.com/htot/linux.git;protocol=https;branch=eds-4.16.0-unified \
         file://ftdi_sio.cfg \
         file://smsc95xx.cfg \
         file://bt_more.cfg \
         file://i2c_chardev.cfg \
         file://configfs.cfg \
+        file://usb_dwc3.cfg \
         "
+SRC_URI_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'acpi', '', 'file://file://i2c_modules.cfg', d)}"
+
+# kernel patches
+SRC_URI_append = "${@bb.utils.contains('DISTRO_FEATURES', 'acpi', '', 'file://0001-Add-kernel-parameter-to-enable-i2c-6-pinctrl-mapping.patch', d)}"
```

Comment:
This one also fails the build - needs a space right after the opening quotation mark, otherwise the content appended is being concatenated with the previous one.

Reply:
Fixed in my commit